Wed May  3 23:19:43 2023
Windows-10-10.0.19045
Ren'Py 8.0.3.22090809
 

Bootstrap to the start of init.init took 0.31s
Early init took 0.01s
Loader init took 0.16s
Loading error handling took 0.13s
Loading script took 2.46s
Loading save slot metadata. took 0.32s
Loading persistent took 0.00s
 - Init at renpy/common/00updater.rpy:23 took 3.91168 s.
Faled to initialize steam: FileNotFoundError("Could not find module 'C:\\Users\\214542\\Desktop\\renpy-8.0.3-sdk\\lib\\py3-windows-x86_64\\steam_api64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
Running init code took 4.06s
Loading analysis data took 0.03s
Analyze and compile ATL took 0.03s
Index archives took 0.00s

Full traceback:
  File "C:\Users\214542\Desktop\renpy-8.0.3-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "C:\Users\214542\Desktop\renpy-8.0.3-sdk\renpy\main.py", line 610, in main
    renpy.dump.dump(False)
  File "C:\Users\214542\Desktop\renpy-8.0.3-sdk\renpy\dump.py", line 262, in dump
    os.rename(new, filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\214542\\Desktop\\renpy-8.0.3-sdk\\Lacuna\\game\\saves\\navigation.json.new' -> 'C:\\Users\\214542\\Desktop\\renpy-8.0.3-sdk\\Lacuna\\game\\saves\\navigation.json'

After initialization, but before game start.
PermissionError: 
Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Intel'"
Renderer: b'Intel(R) UHD Graphics'
Version: b'4.6.0 - Build 30.0.101.1338'
Display Info: None
Screen sizes: virtual=(600, 850) physical=(690, 978) drawable=(690, 978)
Maximum texture size: 4096x4096
